docs/resource-sharing/os-backfill-containers.md

```diff
@@ -94,6 +95,15 @@ but it has other advantages such as supporting automounting of repositories.
 
 ### Adding CVMFS using cvmfsexec
 
+!!! important "EL7 hosts require an updated kernel"
```

Comment:
The updated kernel is most likely already installed everywhere.  The more significant thing I would put in the title is "Unprivileged namespaces must be enabled".  If you don't mention EL7 in the title then hopefully people will also look at it for EL8.

Reply:
Done